keras/utils/training_utils.py

```diff
@@ -101,6 +101,17 @@ def multi_gpu_model(model, gpus):
     if K.backend() != 'tensorflow':
         raise ValueError('`multi_gpu_model` is only available '
                          'with the TensorFlow backend.')
+
+    if not gpus:
+        # Using all visible GPUs when not specifying `gpus`
+        from tensorflow.python.client import device_lib
+
+        tf_devices = device_lib.list_local_devices()
+        gpus = len([x for x in tf_devices if x.device_type == 'GPU'])
```

Comment:
This is not the correct way to list GPUs. You should just call `_get_available_devices`